src/include/verbose.h

```diff
@@ -183,6 +183,13 @@ struct __wt_verbose_multi_category {
 #define __wt_verbose_debug2(session, category, fmt, ...) \
```

Comment:
My preference was to add an additional argument for the VERBOSE DEBUG LEVEL, what are your thoughts @korteland ?

Reply:
I'm not quite sure what you're talking about, sorry. There's a `__wt_verbose_level` macro, but that already includes the level.